src/IssueViz.Security/IssuesStore/IssuesStoreToIssueLocationStoreAdapter.cs

```diff
@@ -0,0 +1,126 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+using SonarLint.VisualStudio.Core.Helpers;
+using SonarLint.VisualStudio.IssueVisualization.Editor.LocationTagging;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.IssuesStore
+{
+    /// <summary>
+    /// Adapter between <see cref="IIssuesStore"/> and <see cref="IIssueLocationStore"/>
+    /// </summary>
+    internal interface IIssuesStoreToIssueLocationStoreAdapter : IIssueLocationStore, IDisposable
+    {
+    }
+
+    /// <summary>
+    /// MEF-imports <see cref="IIssuesStore"/> and implements <see cref="IIssueLocationStore"/> on top of the given stores.
+    /// </summary>
+    [Export(typeof(IIssuesStoreToIssueLocationStoreAdapter))]
+    [Export(typeof(IIssueLocationStore))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    internal sealed class IssuesStoreToIssueLocationStoreAdapter : IIssuesStoreToIssueLocationStoreAdapter
```

Comment:
From the name I'd expect this to be adapter an `IIssueStore` to an `IIssueStoreLocation`; I wouldn't expect it to be an aggregator too.

Possible names that would convey this:
`AggregatingStoreToLocationStoreAdapter`
`AggregatingIssueLocationStoreAdapter`

I don't think it's necessary to include the full source and target type names in the class name; it's pretty obvious from the context and the interfaces that are implemented what the relevant types are.